### PR TITLE
DataTable - Download CSV

### PR DESCRIPTION
**Pull request by titkov49**
Untitled

### CODE REVIEW

**Review by wuujcik — APPROVED** [review]

File: src/containers/DataTable/index.js [hunk 1]
```diff
@@ -36,6 +36,14 @@ export function DataTable ({
 		}, 500);
 	}, [filterValue]);
 
+	const downloadHandler = () => {
+		const list = onDownload();
+		let csv = headers.map(header => header.name).join(',') + "\n" + 
+			list.map(item => headers.map(header => item[header.key]).join(',')).join('\n');
+		let blob = new Blob([csv], {type: "text/csv;charset=utf-8"})
+		saveAs(blob, 'table.csv');
```
Comment (Pe5h4) [inline src/containers/DataTable/index.js:44]: @titkov49 Can be the default name of the table dynamic? e.g. the name of the product based on the `brand_title`, name of the container, where datatable component is used and/or the current datetime? Example:

```
saveAs(blob, `${datetime}_${Config.get('brand_title')}_${containerName}_table.csv');
```

or something with similar composition/order.

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

**Review by ateska — APPROVED** [review]

**Comment by titkov49** [comment]
CHANGES:
- Add dynamic naming to downloaded file (table _title + date in format D-MM-YYYY)

File: src/containers/DataTable/index.js (hunk 1 above)
Comment (titkov49) [reply src/containers/DataTable/index.js]: Added changes

**Review by Pe5h4 — APPROVED** [review]

**Comment by titkov49** [comment]
**Don't merge yet**: 
Download of custom components (https://github.com/TeskaLabs/asab-webui/pull/84) should be added

**Comment by titkov49** [comment]
@wuujcik @Pe5h4

**Review by ateska — APPROVED** [review]

**Review by ateska — APPROVED** [review]

**Comment by titkov49** [comment]
@Pe5h4 https://github.com/TeskaLabs/asab-webui/pull/83/commits/7a4909e3da6be30e6250272334261e4dc415eca0

File: src/containers/DataTable/index.js [hunk 2]
```diff
@@ -59,6 +75,14 @@ export function DataTable ({
 					</Link>
 				</div>
 				}
+				{onDownload &&
+					<div className="float-right pr-3">
+						<Button tag="span" size="sm" onClick={downloadHandler} >
+							<i className="cil-arrow-bottom"></i>
+							Download
```
Comment (Pe5h4) [inline src/containers/DataTable/index.js:82]: @titkov49 Can we implement language localisations to Data table? I mean e.g. in the next iteration/sprint?

**Review by Pe5h4 — APPROVED** [review]

File: src/containers/DataTable/index.js (hunk 2 above)
Comment (titkov49) [reply src/containers/DataTable/index.js:82]: Yep, I have it in current sprint  :)

**Pull request merged**